.credo.exs

```diff
@@ -55,6 +55,7 @@
         {Credo.Check.Warning.UnusedTupleOperation},
         {Credo.Check.Warning.OperationWithConstantResult},
         {CredoEnvvar.Check.Warning.EnvironmentVariablesAtCompileTime},
+        {CredoNaming.Check.Warning.AvoidSpecificTermsInModuleNames, terms: ["Manager", "Fetcher", "Builder", "Persister", "Serializer", "Helper", "Helpers", "Utils"]},
```

Comment:
Is there a way to use a regex to cover all use cases with ending `s`? It would be nice :)

Reply:
The check doesn’t support regular expressions yet but this is a great feature idea. I will implement it soon 👍

Reply:
@Boubalou Your feature is ready! 😉 

https://github.com/mirego/credo_naming/pull/4